app/models/viewmodels/decoder/ROECode.scala

```diff
@@ -43,7 +43,7 @@ object ROECode {
       NoControlRequired,
       RiskingNotPerformed,
       PrelodgePrefix,
-      UnknownRoe
+      UnknownRoe()
```

Comment:
I think you should add here `NoneRoe` too

Reply:
Why?   Its not a code that needs to be decoded

Reply:
Yeah, you're right! My fault

Reply:
If anything, perhaps we should remove 'UnknownRoe' and rename to 'validCodes' or 'kniwnCodes'